common/model/transfer.c

```diff
@@ -427,16 +422,16 @@ int transfer_ctx_count(transfer_ctx_t transfer_ctx) {
 int transfer_ctx_hash(transfer_ctx_t transfer_ctx, Kerl *kerl,
                       transfer_t *transfers, size_t len) {
   size_t i, j, count, current_index = 0;
-  tryte_t essence[TRYTES_PER_ESSENCE];
-  trit_t essence_trits[TRITS_PER_ESSENCE];
+  tryte_t essence[NUM_TRYTES_ESSENCE];

```

Comment:
We really shouldn't be using trytes here... #293

Reply:
agree, but it was not introduced via this PR

Reply:
Yeah, fair enough. That's why I created the issue :)